### PR TITLE
Support for playwright test runner #72

### PR DESCRIPTION
**Pull request by roggenbrot**
Fixes: #72

### CODE REVIEW

**Comment by roggenbrot** [comment]
Any updates?

File: src/runners/PlaywrightTestRunner.ts [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+            this.transformFileName(fileName),
+          ],
+
+        }
+      ],
+      name: "Debug Test",
+      request: "attach",
+      type: "pwa-chrome",
+      port: 9222
+    });*/
```
Comment (felixjb) [inline src/runners/PlaywrightTestRunner.ts:85]: If the code does not do anything useful, I think it should not be commited to the repo.

**Review by felixjb — CHANGES_REQUESTED** [review]

**Comment by felixjb** [comment]
Hi @roggenbrot sorry I took so long to respond. 

How much of the "test runner market share" does playwright owns? I am worried about having to maintain compatibility with many test runners. as it adds a lot of complexity to the next steps of the project.

**Comment by roggenbrot** [comment]
>How much of the "test runner market share" does playwright owns?

playwright test will be replacing jest for e2 testing in the coming months, believe me.

I can give to a demo, or explain you the benefits in details during a chat

>  I am worried about having to maintain compatibility with many test runners

Syntax is equal to jest. So at least for the parser part no real playwright specific changes are required

**Review by felixjb — APPROVED** [review]

**Pull request merged**